package.json

```diff
@@ -29,8 +29,8 @@
   "license": "ISC",
   "repository": "github:ndustrialio/contxt-sdk-js",
   "engines": {
-    "npm": "^5.10.0 ^6.0.0",
-    "node": ">=6"
+    "npm": "^5.6.0",
```

Comment:
Why disallow npm 6 and 7?

Reply:
ah, I meant >, but also, why do we need to restrict on `npm` if `node` is specified?